std_msgs/Header header

PoseWithID[] poses
PointWithID[] points

UnaryFactor[] unary_factors
BinaryFactor[] binary_factors
